pkgs/development/tools/prettierd/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, makeWrapper
+, nodejs
+, fetchYarnDeps
+,
+}:
+mkYarnPackage rec {
+  pname = "prettierd";
+  version = "0.23.4";
+
+  src = fetchFromGitHub {
+    owner = "fsouza";
+    repo = "prettierd";
+    rev = "v${version}";
+    hash = "sha256-GTukjkA/53N9ICdfCJr5HAqhdL5T0pth6zAk8Fu/cis=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/yarn.lock";
+    hash = "sha256-32wMwkVgO5DQuROWnujVGNeCAUq1D6jJurecsD2ROOU=";
+  };
+
+  packageJSON = ./package.json;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildPhase = ''
+    runHook preBuild
+    export HOME=$(mktemp -d)
+    yarn --offline build
+
+    runHook postBuild
+    runHook postBuild
+  '';
```

```suggestion

    runHook postBuild
  '';
```

Same line twice (error when applying the patch I guess).

Remove one instance of `runHook postBuild` please :)